tests/dag_processing/test_manager.py

```diff
@@ -92,7 +92,7 @@ def result(self):
         return self._result
 
     @staticmethod
-    def _fake_dag_processor_factory(file_path, callbacks, dag_ids, pickle_dags):
+    def _create_process(file_path, callbacks, dag_ids, pickle_dags):
```

Comment:
I don't think this is being used anymore, nor is `FakeDagFileProcessorRunner` itself. Might need to patch `DagFileProcessorManager` instead.

Reply:
Yeah. You're right.